bin/internal/update_engine_version.sh

```diff
@@ -54,7 +54,9 @@ if [ -z "$ENGINE_VERSION" ] && [ -f "$FLUTTER_ROOT/DEPS" ] && [ -f "$FLUTTER_ROO
   fi
 fi
 
-if [[ "$BRANCH" != "stable" && "$BRANCH" != "beta" ]]; then
```

Comment:
Does this mean (and in the `ps1` file) we can stop sniffing for `BRANCH` above?

Reply:
Yes! I missed that line a while ago.